apps/tests/aws-runtime/test/tester.test.ts

```diff
@@ -132,7 +132,7 @@ eventualRuntimeTestHarness(
           },
           {
             namespace: "default",
-            n: 6,
+            n: 9,
```

Comment:
Did you mean for this to change

Reply:
yes, I increased the number in the test to try to cause batch events (it worked).